onnxruntime/test/providers/qnn/qnn_test_utils.h

```diff
@@ -220,6 +220,25 @@ void InferenceModel(const std::string& model_data, const char* log_id,
                     ExpectedEPNodeAssignment expected_ep_assignment, const NameMLValMap& feeds,
                     std::vector<OrtValue>& output_vals);
 
+/**
+ * If the ORT_UNIT_TEST_ENABLE_QNN_SAVER environment variable is enabled (set to 1), this function modifies
+ * the QNN EP provider options to enable the QNN Saver backend, which dumps QNN API calls (and weights) to disk.
+ *
+ * - saver_output/saver_output.c: C file containing all QNN API calls.
```

Comment:
>saver_output/saver_output.c [](start = 5, length = 27)

add a UT to verify expected output file generated?

Reply:
Good call. I added unit test for HTP and CPU backends.